client/lib/wp/sync-handler/index.js

```diff
@@ -197,3 +218,7 @@ export class SyncHandler {
 		return localforage.removeItem( key, fn );
 	}
 }
+
+export const pruneRecordsFrom = () => {
+	cacheInvalidation.pruneRecordsFrom( '2 minutes ago' );
```

Comment:
Development purpose. I pushed this branch as soon as was possible to share the approach. That's the reason why I added `[Status] in Progress` and the `[wip]` prefix.
If you set `24 hours ago` you will have to wait one day to see how the records re pruned.